api.planx.uk/modules/slack/controller.ts

```diff
@@ -0,0 +1,46 @@
+import SlackNotify from "slack-notify";
+import { z } from "zod";
+import { ServerError } from "../../errors/index.js";
+import { ValidatedRequestHandler } from "../../shared/middleware/validate.js";
+
+interface SendSlackNotificationResponse {
+  message: string;
+}
+
+export const slackNotificationSchema = z.object({
+  body: z.object({
+    message: z.string(),
+  }),
+});
+
+export type SendSlackNotificationController = ValidatedRequestHandler<
+  typeof slackNotificationSchema,
+  SendSlackNotificationResponse
+>;
+
+export const sendSlackNotifcationController: SendSlackNotificationController =
```

Comment:
```suggestion
export const sendSlackNotificationController: SendSlackNotificationController =
```

An easy word to mispell! 😓